`report_arg_errors`: Span must not be empty and have no suggestion

### Code

```Rust
fn foo() {}
fn main() {
    foo(;
    foo(;
}
```

### Meta
master at eebdfb55fce148676c24555505aebf648123b2de with debug assertions

### Error output

```
error: expected one of `)`, `,`, `.`, `?`, or an operator, found `foo`
 --> src/lib.rs:4:5
  |
3 |     foo(;
  |          -
  |          |
  |          expected one of `)`, `,`, `.`, `?`, or an operator
  |          help: missing `,`
4 |     foo(;
  |     ^^^ unexpected token

error: mismatched closing delimiter: `}`
 --> src/lib.rs:4:8
  |
2 | fn main() {
  |           - closing delimiter possibly meant for this
3 |     foo(;
4 |     foo(;
  |        ^ unclosed delimiter
5 | }
  | ^ mismatched closing delimiter

error[E0061]: this function takes 0 arguments but 1 argument was supplied
 --> src/lib.rs:4:5
  |
4 |     foo(;
  |     ^^^
5 | }
  | -
  | |
  | unexpected argument
  | help: remove the extra argument
  |
note: function defined here
 --> src/lib.rs:1:4
  |
1 | fn foo() {}
  |    ^^^

thread 'rustc' panicked at 'assertion failed: `(left == right)`
  left: `Some(SubstitutionPart { span: src/lib.rs:4:5: 4:5 (#0), snippet: "" })`,
 right: `None`: Span must not be empty and have no suggestion', /home/lukas/code/rust/compiler/rustc_errors/src/diagnostic.rs:627:9
```

<details><summary><strong>Backtrace</strong></summary>
<p>

```
stack backtrace:
   0:     0x7fc844ec5507 - std::backtrace_rs::backtrace::libunwind::trace::hb0876bb39e7adf0d
                               at /home/lukas/code/rust/library/std/src/../../backtrace/src/backtrace/libunwind.rs:93:5
   1:     0x7fc844ec5507 - std::backtrace_rs::backtrace::trace_unsynchronized::h1bcab1313827007b
                               at /home/lukas/code/rust/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
   2:     0x7fc844eaab27 - std::sys_common::backtrace::_print_fmt::h262d2a62279fa102
                               at /home/lukas/code/rust/library/std/src/sys_common/backtrace.rs:65:5
   3:     0x7fc844eaab27 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h6bd5257e275ad72e
                               at /home/lukas/code/rust/library/std/src/sys_common/backtrace.rs:44:22
   4:     0x7fc844f5b449 - core::fmt::write::hfc2320e8ddcd13ea
                               at /home/lukas/code/rust/library/core/src/fmt/mod.rs:1232:17
   5:     0x7fc844edfdd1 - std::io::Write::write_fmt::hbf41212d1353dab3
                               at /home/lukas/code/rust/library/std/src/io/mod.rs:1684:15
   6:     0x7fc844eaa98b - std::sys_common::backtrace::_print::h04bffdca3a2e4507
                               at /home/lukas/code/rust/library/std/src/sys_common/backtrace.rs:47:5
   7:     0x7fc844eaa98b - std::sys_common::backtrace::print::h19fcbd1cf9bfb9fa
                               at /home/lukas/code/rust/library/std/src/sys_common/backtrace.rs:34:9
   8:     0x7fc844ec4407 - std::panicking::default_hook::{{closure}}::hd7a322cb7e0d374e
   9:     0x7fc844ec417b - std::panicking::default_hook::hcbf6f128d636bde9
                               at /home/lukas/code/rust/library/std/src/panicking.rs:286:9
  10:     0x7fc845be364f - rustc_driver_impl[6b0b459770ccd82f]::DEFAULT_HOOK::{closure#0}::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_driver_impl/src/lib.rs:1145:17
  11:     0x7fc844ec4af4 - std::panicking::rust_panic_with_hook::h8021e3346936b8f5
                               at /home/lukas/code/rust/library/std/src/panicking.rs:692:13
  12:     0x7fc844eb4fa3 - std::panicking::begin_panic_handler::{{closure}}::hd30d002ab4f44e6b
                               at /home/lukas/code/rust/library/std/src/panicking.rs:579:13
  13:     0x7fc844eb4edc - std::sys_common::backtrace::__rust_end_short_backtrace::h602bbc1d94975c5f
                               at /home/lukas/code/rust/library/std/src/sys_common/backtrace.rs:137:18
  14:     0x7fc844ec44e1 - rust_begin_unwind
                               at /home/lukas/code/rust/library/std/src/panicking.rs:575:5
  15:     0x7fc844e7fb03 - core::panicking::panic_fmt::hd1c823890d7b12ad
                               at /home/lukas/code/rust/library/core/src/panicking.rs:64:14
  16:     0x7fc844e7fe69 - core::panicking::assert_failed_inner::hea91b652e4ebf3c4
  17:     0x7fc8459b1e5b - core[861a0f7f61dec426]::panicking::assert_failed::<core[861a0f7f61dec426]::option::Option<&rustc_errors[1b63d2671a744be7]::SubstitutionPart>, core[861a0f7f61dec426]::option::Option<&rustc_errors[1b63d2671a744be7]::SubstitutionPart>>
                               at /home/lukas/code/rust/library/core/src/panicking.rs:211:5
  18:     0x7fc846177536 - <rustc_errors[1b63d2671a744be7]::diagnostic::Diagnostic>::multipart_suggestion_with_style::<&alloc[508b0f1a3af6c651]::string::String>
                               at /home/lukas/code/rust/compiler/rustc_errors/src/diagnostic.rs:627:9
  19:     0x7fc846173ed6 - <rustc_errors[1b63d2671a744be7]::diagnostic::Diagnostic>::multipart_suggestion::<&alloc[508b0f1a3af6c651]::string::String>
                               at /home/lukas/code/rust/compiler/rustc_errors/src/diagnostic.rs:588:9
  20:     0x7fc8461d1d34 - <rustc_errors[1b63d2671a744be7]::diagnostic_builder::DiagnosticBuilder<rustc_errors[1b63d2671a744be7]::ErrorGuaranteed>>::multipart_suggestion::<&alloc[508b0f1a3af6c651]::string::String>
                               at /home/lukas/code/rust/compiler/rustc_errors/src/diagnostic_builder.rs:481:13
  21:     0x7fc846208fd8 - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::report_arg_errors
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/fn_ctxt/checks.rs:1194:17
  22:     0x7fc846204ed7 - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_argument_types
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/fn_ctxt/checks.rs:450:13
  23:     0x7fc8461dafa8 - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::confirm_builtin_call
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/callee.rs:450:9
  24:     0x7fc8461d9bec - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_call
  25:     0x7fc846248c09 - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_expr_kind
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/expr.rs:329:45
  26:     0x7fc8461ec25a - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_expr_with_expectation_and_args::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/expr.rs:235:18
  27:     0x7fc8461ec25a - stacker[86b279fcb4e335a2]::maybe_grow::<rustc_middle[aae142def08bf2ab]::ty::Ty, <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_expr_with_expectation_and_args::{closure#0}>
                               at /home/lukas/.cargo/registry/src/github.com-1ecc6299db9ec823/stacker-0.1.15/src/lib.rs:55:9
  28:     0x7fc8461ec25a - rustc_data_structures[db7360aa580e6a43]::stack::ensure_sufficient_stack::<rustc_middle[aae142def08bf2ab]::ty::Ty, <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_expr_with_expectation_and_args::{closure#0}>
                               at /home/lukas/code/rust/compiler/rustc_data_structures/src/stack.rs:17:5
  29:     0x7fc8461ec25a - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_expr_with_expectation_and_args
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/expr.rs:231:18
  30:     0x7fc846247dfd - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_expr_with_expectation
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/expr.rs:185:9
  31:     0x7fc8460d2d89 - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_block_with_expected::{closure#0}::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/fn_ctxt/checks.rs:1530:50
  32:     0x7fc8460d2d89 - <core[861a0f7f61dec426]::option::Option<&&rustc_hir[18e8316ed79a758d]::hir::Expr>>::map::<rustc_middle[aae142def08bf2ab]::ty::Ty, <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_block_with_expected::{closure#0}::{closure#0}>
                               at /home/lukas/code/rust/library/core/src/option.rs:972:29
  33:     0x7fc8460d2d89 - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_block_with_expected::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/fn_ctxt/checks.rs:1530:32
  34:     0x7fc8460d2d89 - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::with_breakable_ctxt::<<rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_block_with_expected::{closure#0}, ()>
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/fn_ctxt/_impl.rs:1439:22
  35:     0x7fc84620c485 - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_block_with_expected
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/fn_ctxt/checks.rs:1523:26
  36:     0x7fc846248dd8 - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_expr_kind
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/expr.rs:328:41
  37:     0x7fc8461ec25a - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_expr_with_expectation_and_args::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/expr.rs:235:18
  38:     0x7fc8461ec25a - stacker[86b279fcb4e335a2]::maybe_grow::<rustc_middle[aae142def08bf2ab]::ty::Ty, <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_expr_with_expectation_and_args::{closure#0}>
                               at /home/lukas/.cargo/registry/src/github.com-1ecc6299db9ec823/stacker-0.1.15/src/lib.rs:55:9
  39:     0x7fc8461ec25a - rustc_data_structures[db7360aa580e6a43]::stack::ensure_sufficient_stack::<rustc_middle[aae142def08bf2ab]::ty::Ty, <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_expr_with_expectation_and_args::{closure#0}>
                               at /home/lukas/code/rust/compiler/rustc_data_structures/src/stack.rs:17:5
  40:     0x7fc8461ec25a - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_expr_with_expectation_and_args
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/expr.rs:231:18
  41:     0x7fc846247dfd - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_expr_with_expectation
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/expr.rs:185:9
  42:     0x7fc8461ee059 - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_expr_with_hint
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/expr.rs:137:9
  43:     0x7fc8461ee059 - <rustc_hir_typeck[cca5ec3204630dfb]::fn_ctxt::FnCtxt>::check_return_expr
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/expr.rs:835:30
  44:     0x7fc84626ce91 - rustc_hir_typeck[cca5ec3204630dfb]::check::check_fn
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/check.rs:122:9
  45:     0x7fc84607bf60 - rustc_hir_typeck[cca5ec3204630dfb]::typeck_with_fallback::<rustc_hir_typeck[cca5ec3204630dfb]::typeck::{closure#0}>::{closure#0}::{closure#1}
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/lib.rs:219:13
  46:     0x7fc84607bf60 - <rustc_hir_typeck[cca5ec3204630dfb]::inherited::InheritedBuilder>::enter::<rustc_hir_typeck[cca5ec3204630dfb]::typeck_with_fallback<rustc_hir_typeck[cca5ec3204630dfb]::typeck::{closure#0}>::{closure#0}::{closure#1}, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/inherited.rs:108:9
  47:     0x7fc8462fb051 - rustc_hir_typeck[cca5ec3204630dfb]::typeck_with_fallback::<rustc_hir_typeck[cca5ec3204630dfb]::typeck::{closure#0}>::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/lib.rs:202:26
  48:     0x7fc8462fb051 - rustc_hir_typeck[cca5ec3204630dfb]::typeck_with_fallback::<rustc_hir_typeck[cca5ec3204630dfb]::typeck::{closure#0}>
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/lib.rs:185:36
  49:     0x7fc8461cef95 - rustc_hir_typeck[cca5ec3204630dfb]::typeck
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/lib.rs:166:9
  50:     0x7fc84761c537 - <rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl::<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, rustc_span[7230cd2509a336a2]::def_id::LocalDefId, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_query_system/src/dep_graph/graph.rs:340:53
  51:     0x7fc84761c537 - rustc_middle[aae142def08bf2ab]::ty::context::tls::tlv::with_tlv::<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, rustc_span[7230cd2509a336a2]::def_id::LocalDefId, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:94:9
  52:     0x7fc84761c537 - rustc_middle[aae142def08bf2ab]::ty::context::tls::enter_context::<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, rustc_span[7230cd2509a336a2]::def_id::LocalDefId, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:114:5
  53:     0x7fc84761c537 - <rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind as rustc_query_system[562202d82219ff72]::dep_graph::DepKind>::with_deps::<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, rustc_span[7230cd2509a336a2]::def_id::LocalDefId, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_middle/src/dep_graph/mod.rs:58:13
  54:     0x7fc84761c537 - rustc_middle[aae142def08bf2ab]::ty::context::tls::with_context::<<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind as rustc_query_system[562202d82219ff72]::dep_graph::DepKind>::with_deps<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, rustc_span[7230cd2509a336a2]::def_id::LocalDefId, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:142:36
  55:     0x7fc84761c537 - rustc_middle[aae142def08bf2ab]::ty::context::tls::with_context_opt::<rustc_middle[aae142def08bf2ab]::ty::context::tls::with_context<<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind as rustc_query_system[562202d82219ff72]::dep_graph::DepKind>::with_deps<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, rustc_span[7230cd2509a336a2]::def_id::LocalDefId, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:131:18
  56:     0x7fc84761c537 - rustc_middle[aae142def08bf2ab]::ty::context::tls::with_context::<<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind as rustc_query_system[562202d82219ff72]::dep_graph::DepKind>::with_deps<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, rustc_span[7230cd2509a336a2]::def_id::LocalDefId, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:142:5
  57:     0x7fc84761c537 - <rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind as rustc_query_system[562202d82219ff72]::dep_graph::DepKind>::with_deps::<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, rustc_span[7230cd2509a336a2]::def_id::LocalDefId, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>::{closure#0}, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/dep_graph/mod.rs:55:9
  58:     0x7fc847723008 - <rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl::<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, rustc_span[7230cd2509a336a2]::def_id::LocalDefId, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>
                               at /home/lukas/code/rust/compiler/rustc_query_system/src/dep_graph/graph.rs:340:22
  59:     0x7fc847723008 - <rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task::<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, rustc_span[7230cd2509a336a2]::def_id::LocalDefId, &rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults>
                               at /home/lukas/code/rust/compiler/rustc_query_system/src/dep_graph/graph.rs:290:13
  60:     0x7fc84791dc11 - rustc_query_system[562202d82219ff72]::query::plumbing::execute_job::<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}
                               at /home/lukas/code/rust/compiler/rustc_query_system/src/query/plumbing.rs:463:13
  61:     0x7fc84791dc11 - stacker[86b279fcb4e335a2]::maybe_grow::<(&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex), rustc_query_system[562202d82219ff72]::query::plumbing::execute_job<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}>
                               at /home/lukas/.cargo/registry/src/github.com-1ecc6299db9ec823/stacker-0.1.15/src/lib.rs:55:9
  62:     0x7fc84791dc11 - rustc_data_structures[db7360aa580e6a43]::stack::ensure_sufficient_stack::<(&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex), rustc_query_system[562202d82219ff72]::query::plumbing::execute_job<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}>
                               at /home/lukas/code/rust/compiler/rustc_data_structures/src/stack.rs:17:5
  63:     0x7fc84791dc11 - <rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt as rustc_query_system[562202d82219ff72]::query::QueryContext>::start_query::<(&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex), rustc_query_system[562202d82219ff72]::query::plumbing::execute_job<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}>::{closure#0}::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_query_impl/src/plumbing.rs:128:17
  64:     0x7fc84791dc11 - rustc_middle[aae142def08bf2ab]::ty::context::tls::tlv::with_tlv::<<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt as rustc_query_system[562202d82219ff72]::query::QueryContext>::start_query<(&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex), rustc_query_system[562202d82219ff72]::query::plumbing::execute_job<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}>::{closure#0}::{closure#0}, (&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex)>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:94:9
  65:     0x7fc84791dc11 - rustc_middle[aae142def08bf2ab]::ty::context::tls::enter_context::<<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt as rustc_query_system[562202d82219ff72]::query::QueryContext>::start_query<(&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex), rustc_query_system[562202d82219ff72]::query::plumbing::execute_job<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}>::{closure#0}::{closure#0}, (&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex)>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:114:5
  66:     0x7fc84791dc11 - <rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt as rustc_query_system[562202d82219ff72]::query::QueryContext>::start_query::<(&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex), rustc_query_system[562202d82219ff72]::query::plumbing::execute_job<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}>::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_query_impl/src/plumbing.rs:127:13
  67:     0x7fc84791dc11 - rustc_middle[aae142def08bf2ab]::ty::context::tls::with_related_context::<<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt as rustc_query_system[562202d82219ff72]::query::QueryContext>::start_query<(&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex), rustc_query_system[562202d82219ff72]::query::plumbing::execute_job<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}>::{closure#0}, (&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex)>::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:164:9
  68:     0x7fc84791dc11 - rustc_middle[aae142def08bf2ab]::ty::context::tls::with_context::<rustc_middle[aae142def08bf2ab]::ty::context::tls::with_related_context<<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt as rustc_query_system[562202d82219ff72]::query::QueryContext>::start_query<(&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex), rustc_query_system[562202d82219ff72]::query::plumbing::execute_job<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}>::{closure#0}, (&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex)>::{closure#0}, (&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex)>::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:142:36
  69:     0x7fc84791dc11 - rustc_middle[aae142def08bf2ab]::ty::context::tls::with_context_opt::<rustc_middle[aae142def08bf2ab]::ty::context::tls::with_context<rustc_middle[aae142def08bf2ab]::ty::context::tls::with_related_context<<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt as rustc_query_system[562202d82219ff72]::query::QueryContext>::start_query<(&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex), rustc_query_system[562202d82219ff72]::query::plumbing::execute_job<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}>::{closure#0}, (&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex)>::{closure#0}, (&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex)>::{closure#0}, (&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex)>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:131:18
  70:     0x7fc84791dc11 - rustc_middle[aae142def08bf2ab]::ty::context::tls::with_context::<rustc_middle[aae142def08bf2ab]::ty::context::tls::with_related_context<<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt as rustc_query_system[562202d82219ff72]::query::QueryContext>::start_query<(&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex), rustc_query_system[562202d82219ff72]::query::plumbing::execute_job<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}>::{closure#0}, (&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex)>::{closure#0}, (&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex)>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:142:5
  71:     0x7fc84791dc11 - rustc_middle[aae142def08bf2ab]::ty::context::tls::with_related_context::<<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt as rustc_query_system[562202d82219ff72]::query::QueryContext>::start_query<(&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex), rustc_query_system[562202d82219ff72]::query::plumbing::execute_job<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}>::{closure#0}, (&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex)>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:155:5
  72:     0x7fc84791dc11 - <rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt as rustc_query_system[562202d82219ff72]::query::QueryContext>::start_query::<(&rustc_middle[aae142def08bf2ab]::ty::typeck_results::TypeckResults, rustc_query_system[562202d82219ff72]::dep_graph::graph::DepNodeIndex), rustc_query_system[562202d82219ff72]::query::plumbing::execute_job<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}>
                               at /home/lukas/code/rust/compiler/rustc_query_impl/src/plumbing.rs:112:9
  73:     0x7fc84791dc11 - rustc_query_system[562202d82219ff72]::query::plumbing::execute_job::<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>
                               at /home/lukas/code/rust/compiler/rustc_query_system/src/query/plumbing.rs:453:9
  74:     0x7fc84791dc11 - rustc_query_system[562202d82219ff72]::query::plumbing::try_execute_query::<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>
                               at /home/lukas/code/rust/compiler/rustc_query_system/src/query/plumbing.rs:364:17
  75:     0x7fc8479a1a90 - rustc_query_system[562202d82219ff72]::query::plumbing::get_query::<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>
                               at /home/lukas/code/rust/compiler/rustc_query_system/src/query/plumbing.rs:743:36
  76:     0x7fc847a7d024 - <rustc_query_impl[d8c5c00af94eda8c]::Queries as rustc_middle[aae142def08bf2ab]::ty::query::QueryEngine>::typeck::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_query_impl/src/plumbing.rs:744:17
  77:     0x7fc847a7d024 - <rustc_query_impl[d8c5c00af94eda8c]::Queries as rustc_middle[aae142def08bf2ab]::ty::query::QueryEngine>::typeck
                               at /home/lukas/code/rust/compiler/rustc_query_impl/src/lib.rs:46:1
  78:     0x7fc8462e449c - <rustc_middle[aae142def08bf2ab]::ty::query::TyCtxtEnsure>::typeck::<rustc_span[7230cd2509a336a2]::def_id::LocalDefId>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/query.rs:284:29
  79:     0x7fc8462e449c - rustc_hir_typeck[cca5ec3204630dfb]::typeck_item_bodies::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/lib.rs:150:51
  80:     0x7fc8462e449c - <rustc_middle[aae142def08bf2ab]::hir::map::Map>::par_body_owners::<rustc_hir_typeck[cca5ec3204630dfb]::typeck_item_bodies::{closure#0}>::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_middle/src/hir/map/mod.rs:491:82
  81:     0x7fc8462e449c - rustc_data_structures[db7360aa580e6a43]::sync::par_for_each_in::<&[rustc_span[7230cd2509a336a2]::def_id::LocalDefId], <rustc_middle[aae142def08bf2ab]::hir::map::Map>::par_body_owners<rustc_hir_typeck[cca5ec3204630dfb]::typeck_item_bodies::{closure#0}>::{closure#0}>::{closure#0}::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_data_structures/src/sync.rs:152:66
  82:     0x7fc8462e449c - <core[861a0f7f61dec426]::panic::unwind_safe::AssertUnwindSafe<rustc_data_structures[db7360aa580e6a43]::sync::par_for_each_in<&[rustc_span[7230cd2509a336a2]::def_id::LocalDefId], <rustc_middle[aae142def08bf2ab]::hir::map::Map>::par_body_owners<rustc_hir_typeck[cca5ec3204630dfb]::typeck_item_bodies::{closure#0}>::{closure#0}>::{closure#0}::{closure#0}> as core[861a0f7f61dec426]::ops::function::FnOnce<()>>::call_once
                               at /home/lukas/code/rust/library/core/src/panic/unwind_safe.rs:271:9
  83:     0x7fc8462eb5bb - std[bb1efaaece8fc66c]::panicking::try::do_call::<core[861a0f7f61dec426]::panic::unwind_safe::AssertUnwindSafe<rustc_data_structures[db7360aa580e6a43]::sync::par_for_each_in<&[rustc_span[7230cd2509a336a2]::def_id::LocalDefId], <rustc_middle[aae142def08bf2ab]::hir::map::Map>::par_body_owners<rustc_hir_typeck[cca5ec3204630dfb]::typeck_item_bodies::{closure#0}>::{closure#0}>::{closure#0}::{closure#0}>, ()>
                               at /home/lukas/code/rust/library/std/src/panicking.rs:483:40
  84:     0x7fc8462eb5bb - std[bb1efaaece8fc66c]::panicking::try::<(), core[861a0f7f61dec426]::panic::unwind_safe::AssertUnwindSafe<rustc_data_structures[db7360aa580e6a43]::sync::par_for_each_in<&[rustc_span[7230cd2509a336a2]::def_id::LocalDefId], <rustc_middle[aae142def08bf2ab]::hir::map::Map>::par_body_owners<rustc_hir_typeck[cca5ec3204630dfb]::typeck_item_bodies::{closure#0}>::{closure#0}>::{closure#0}::{closure#0}>>
                               at /home/lukas/code/rust/library/std/src/panicking.rs:447:19
  85:     0x7fc8460781a4 - rustc_data_structures[db7360aa580e6a43]::sync::par_for_each_in::<&[rustc_span[7230cd2509a336a2]::def_id::LocalDefId], <rustc_middle[aae142def08bf2ab]::hir::map::Map>::par_body_owners<rustc_hir_typeck[cca5ec3204630dfb]::typeck_item_bodies::{closure#0}>::{closure#0}>::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_data_structures/src/sync.rs:152:33
  86:     0x7fc8460781a4 - <core[861a0f7f61dec426]::slice::iter::Iter<rustc_span[7230cd2509a336a2]::def_id::LocalDefId> as core[861a0f7f61dec426]::iter::traits::iterator::Iterator>::for_each::<rustc_data_structures[db7360aa580e6a43]::sync::par_for_each_in<&[rustc_span[7230cd2509a336a2]::def_id::LocalDefId], <rustc_middle[aae142def08bf2ab]::hir::map::Map>::par_body_owners<rustc_hir_typeck[cca5ec3204630dfb]::typeck_item_bodies::{closure#0}>::{closure#0}>::{closure#0}>
                               at /home/lukas/code/rust/library/core/src/slice/iter/macros.rs:201:21
  87:     0x7fc8460781a4 - rustc_data_structures[db7360aa580e6a43]::sync::par_for_each_in::<&[rustc_span[7230cd2509a336a2]::def_id::LocalDefId], <rustc_middle[aae142def08bf2ab]::hir::map::Map>::par_body_owners<rustc_hir_typeck[cca5ec3204630dfb]::typeck_item_bodies::{closure#0}>::{closure#0}>
                               at /home/lukas/code/rust/compiler/rustc_data_structures/src/sync.rs:151:13
  88:     0x7fc8461cedd3 - <rustc_middle[aae142def08bf2ab]::hir::map::Map>::par_body_owners::<rustc_hir_typeck[cca5ec3204630dfb]::typeck_item_bodies::{closure#0}>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/hir/map/mod.rs:491:9
  89:     0x7fc8461cedd3 - rustc_hir_typeck[cca5ec3204630dfb]::typeck_item_bodies
                               at /home/lukas/code/rust/compiler/rustc_hir_typeck/src/lib.rs:150:5
  90:     0x7fc847621d43 - <rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl::<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, (), ()>::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_query_system/src/dep_graph/graph.rs:340:53
  91:     0x7fc847621d43 - rustc_middle[aae142def08bf2ab]::ty::context::tls::tlv::with_tlv::<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, (), ()>::{closure#0}, ()>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:94:9
  92:     0x7fc847621d43 - rustc_middle[aae142def08bf2ab]::ty::context::tls::enter_context::<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, (), ()>::{closure#0}, ()>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:114:5
  93:     0x7fc847621d43 - <rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind as rustc_query_system[562202d82219ff72]::dep_graph::DepKind>::with_deps::<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, (), ()>::{closure#0}, ()>::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_middle/src/dep_graph/mod.rs:58:13
  94:     0x7fc847621d43 - rustc_middle[aae142def08bf2ab]::ty::context::tls::with_context::<<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind as rustc_query_system[562202d82219ff72]::dep_graph::DepKind>::with_deps<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, (), ()>::{closure#0}, ()>::{closure#0}, ()>::{closure#0}
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:142:36
  95:     0x7fc847621d43 - rustc_middle[aae142def08bf2ab]::ty::context::tls::with_context_opt::<rustc_middle[aae142def08bf2ab]::ty::context::tls::with_context<<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind as rustc_query_system[562202d82219ff72]::dep_graph::DepKind>::with_deps<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, (), ()>::{closure#0}, ()>::{closure#0}, ()>::{closure#0}, ()>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:131:18
  96:     0x7fc847621d43 - rustc_middle[aae142def08bf2ab]::ty::context::tls::with_context::<<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind as rustc_query_system[562202d82219ff72]::dep_graph::DepKind>::with_deps<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, (), ()>::{closure#0}, ()>::{closure#0}, ()>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/ty/context/tls.rs:142:5
  97:     0x7fc847621d43 - <rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind as rustc_query_system[562202d82219ff72]::dep_graph::DepKind>::with_deps::<<rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, (), ()>::{closure#0}, ()>
                               at /home/lukas/code/rust/compiler/rustc_middle/src/dep_graph/mod.rs:55:9
  98:     0x7fc847770033 - <rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task_impl::<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, (), ()>
                               at /home/lukas/code/rust/compiler/rustc_query_system/src/dep_graph/graph.rs:340:22
  99:     0x7fc847770033 - <rustc_query_system[562202d82219ff72]::dep_graph::graph::DepGraph<rustc_middle[aae142def08bf2ab]::dep_graph::dep_node::DepKind>>::with_task::<rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt, (), ()>
                               at /home/lukas/code/rust/compiler/rustc_query_system/src/dep_graph/graph.rs:290:13
 100:     0x7fc8478940fc - rustc_query_system[562202d82219ff72]::query::plumbing::execute_job::<rustc_query_impl[d8c5c00af94eda8c]::queries::typeck_item_bodies, rustc_query_impl[d8c5c00af94eda8c]::plumbing::QueryCtxt>::{closure#4}
                               at /home/lukas/code/rust/compiler/rustc_query_system/src/query/plumbing.rs:463:13

[Comment is too long (maximum is 65536 characters)]

error: the compiler unexpectedly panicked. this is a bug.

note: we would appreciate a bug report: https://github.com/rust-lang/rust/issues/new?labels=C-bug%2C+I-ICE%2C+T-compiler&template=ice.md

note: rustc 1.69.0-dev running on x86_64-unknown-linux-gnu

note: compiler flags: --crate-type lib -C embed-bitcode=no -C debuginfo=2 -C incremental=[REDACTED]

note: some of the compiler flags provided by cargo are hidden

query stack during panic:
#0 [typeck] type-checking `main`
#1 [typeck_item_bodies] type-checking all item bodies
#2 [analysis] running analysis passes on this crate
end of query stack
```

</p>
</details>

https://github.com/rust-lang/rust/blob/eebdfb55fce148676c24555505aebf648123b2de/compiler/rustc_hir_typeck/src/fn_ctxt/checks.rs#L1194

Related to https://github.com/rust-lang/rust/issues/108225, but not fixed by https://github.com/rust-lang/rust/pull/108239.

@rustbot label requires-debug-assertions S-bug-has-mcve

--- a/tests/ui/parser/issues/issue-108242-semicolon-recovery.rs
+++ b/tests/ui/parser/issues/issue-108242-semicolon-recovery.rs
@@ -1,0 +1,5 @@
+fn foo() {}
+fn main() {
+    foo(;
+    foo(;
+} //~ ERROR mismatched closing delimiter

--- a/tests/ui/parser/issues/issue-108242-semicolon-recovery.stderr
+++ b/tests/ui/parser/issues/issue-108242-semicolon-recovery.stderr
@@ -1,0 +1,13 @@
+error: mismatched closing delimiter: `}`
+  --> $DIR/issue-108242-semicolon-recovery.rs:4:8
+   |
+LL | fn main() {
+   |           - closing delimiter possibly meant for this
+LL |     foo(;
+LL |     foo(;
+   |        ^ unclosed delimiter
+LL | }
+   | ^ mismatched closing delimiter
+
+error: aborting due to previous error
+

